### ISSUE TITLE
Taskbar icon

### ISSUE THREAD

**Issue by smadgerano** [bergerpascal/MSIX_Commander#4]
The icon in the taskbar is for PowerShell, not MSIX Commander

**Comment by bergerpascal** [comment]
Yes unfortunately. The Tool is written in PowerShell and then compiled to an exe. There was an issue when i crated the MSIX with that exe that some of the features where not working that way, so i had to create an other small PowerShell script(exe) to launch the main script and therefore it will show the PowerShell icon.
